15-7-1_and_15-c-1/15-7-1_and_15-c-1_checks.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "15-7-1 and 15-c-1 checks"
+author: "Katie Uzzell"
+date: "22/12/2022"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r load-packages, include=FALSE}
+library(tidyr)
+library(dplyr)
+library(ggplot2)
+library(kableExtra)
+```
+
+```{r include= FALSE}
+source('15-7-1_15-c-1_example_config.R')
+
+old_data <- read.csv("https://sdgdata.gov.uk/sdg-data/en/data/15-7-1.csv")
+
+date <- Sys.Date()
+new_data <- read.csv(paste0("Example_Output/", date, "_update_15-7-1_15-c-1", ".csv")) 
```

Comment:
"Example_Output shouldn't be hard-coded here. Try instead:
`new_data <- read.csv(paste0(output_folder, "/", date, "_update_15-7-1_15-c-1", ".csv"))`

Reply:
Done